README.md

```diff
@@ -158,7 +158,7 @@ Flags:
 
 * `--builderrc`: Path to builder config file (default: `.builderrc`)
 * `--tries`: Number of times to attempt a task (default: `1`)
-* `--setup`: Single task to run for the entirety of <action>.
+* `--setup`: Single task to run for the entirety of `<action>`.
```

Comment:
Without the backticks, the word `<action>` wasn't showing up at all!
![screen shot 2015-12-20 at 4 20 34 pm](https://cloud.githubusercontent.com/assets/6352327/11921103/0f0d62c6-a736-11e5-8fd2-fa8d66cf8157.png)